src/main/java/seedu/address/model/Model.java

```diff
@@ -84,4 +84,9 @@ public interface Model {
      * @throws NullPointerException if {@code predicate} is null.
      */
     void updateFilteredClientList(Predicate<Client> predicate);
+
+    /**
+     * Updates the total transaction of the filtered client list
```

Comment:
You might have missed out a fullstop on this javadocs.

Reply:
Apologies I have added this as well. Thank you